### DISCUSSION TITLE
FMOD

### DISCUSSION THREAD

**Discussion by EvilGeniusTC** [MonoGame/MonoGame#8395] [Audio]
![Screenshot 2024-06-22 174602](https://github.com/MonoGame/MonoGame/assets/136780698/32a8a030-bf9d-4e61-83a8-bc6ff1bb2a73)
I got FmodforFoxes installed and all the dlls in the correct place . when i get to the point where i paste the code into my initialiazation method it doesnt work because im in the wrong namespace but i cant have both monogame and fmodfoxes namespaces so im confused.

**Comment by tomspilman** [comment]
You can always use the fill namespace like `FmodForFoxes.FMODMode.CoreAndStudio` to resolve namespace conflicts if that is the issue.

Here it just seems like your using the `using FmodForFoxes;`.
